network-generator/src/main/resources/kotlin-ktor-client/api.mustache

```diff
@@ -36,6 +37,14 @@ import io.ktor.http.path
     */
     {{^nonPublicApi}}public {{/nonPublicApi}}suspend fun {{operationId}}({{#allParams}}{{paramName}}: {{{dataType}}}{{^required}}? = null{{/required}}{{^-last}}, {{/-last}}{{/allParams}}) : {{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}kotlin.Unit{{/returnType}}
 
+    /**
+    * {{summary}}
+    * {{notes}}
+    {{#allParams}}* @param {{paramName}} {{description}} {{^required}}(optional{{#defaultValue}}, default to {{{.}}}{{/defaultValue}}){{/required}}
+    {{/allParams}}* @return HttpResponse
+    */
+    {{^nonPublicApi}}public {{/nonPublicApi}}suspend fun {{operationId}}Response({{#allParams}}{{paramName}}: {{{dataType}}}{{^required}}? = null{{/required}}{{^-last}}, {{/-last}}{{/allParams}}) : HttpResponse
```

Comment:
we should use generic response wrapper.
something like:
```
data class NetworkResponse<T>(
    val httpResponse: HttpResponse,
    val body: T
)
```

Reply:
I added, please see the changes 😎